app/controllers/guide_controller.rb

```diff
@@ -1,11 +1,11 @@
 class GuideController < ApplicationController
   include ApiRedirectable
-  include Previewable
   include Cacheable
```

Comment:
to follow on from that: you think we should try and replicate that functionality in draft-frontend? i.e. not set a future expiry + maybe set no-cache